udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDatasetsTable/AdminDatasetsTable.vue

```diff
@@ -0,0 +1,216 @@
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ formatDate(dataset.created_at) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ formatDate(dataset.last_modified) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ getFilesCount(dataset) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          <QualityScoreTooltip :dataset="dataset"/>
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.discussions) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.views) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.resources_downloads ?? 0) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.reuses) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.followers) }}
+        </AdminTableTd>
+      </tr>
+    </tbody>
+  </AdminTable>
+</template>
+<script setup lang="ts">
+import { formatDate } from "@datagouv/components";
+import type { Dataset, DatasetV2 } from '@datagouv/components';
+import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import AdminBadge from "../../AdminBadge/AdminBadge.vue";
+import AdminContentWithTooltip from "../../AdminContentWithTooltip/AdminContentWithTooltip.vue";
+import AdminTable from "../Table/AdminTable.vue";
+import AdminTableTd from "../Table/AdminTableTd.vue";
+import AdminTableTh from "../Table/AdminTableTh.vue";
+import Tooltip from '../../Tooltip/Tooltip.vue';
+import QualityScoreTooltip from "../../dataset/QualityScore/QualityScoreTooltip/QualityScoreTooltip.vue";
+import { admin_root } from '../../../config';
+import { summarize } from '../../../helpers';
+import type { AdminBadgeState, DatasetSortedBy, SortDirection } from '../../../types';
+
+const emit = defineEmits<{
+  (event: 'sort', sort: string): void
+}>();
+
+defineProps<{
+  datasets: Array<Dataset | DatasetV2>
+}>();
+
+const { t } = useI18n();
+
+const sort = ref<SortDirection>('desc');
+const sortedBy = ref<DatasetSortedBy>('created');
+
+type DatasetStatuses = "deleted" | "archived" | "private" | "public";
+
+const datasetStatusBadge: Record<DatasetStatuses, {label: string, type: AdminBadgeState}> = {
+  deleted: {
+    label: t("Deleted"),
+    type: "error",
+  },
+  archived: {
+    label: t("Archived"),
+    type: "warning",
+  },
+  private: {
+    label: t("Private"),
+    type: "default",
+  },
+  public: {
+    label: t("Public"),
+    type: "info"
+  }
+}
+
+function updateSort(column: DatasetSortedBy, direction: SortDirection) {
+  sort.value = direction;
+  sortedBy.value = column;
+  emit('sort', `${sort.value === 'asc' ? "" : "-"}${sortedBy.value}`);
+}
+
+function sorted(column: DatasetSortedBy) {
+  if(sortedBy.value === column) {
+    return sort.value;
+  }
+  return undefined;
```

Comment:
I prefer to use `null` when it's a valid response (for me `undefined` is an error in JS, something we forget couldn't be set… `null` is set as `null` for a valid reason, not sure if it's clear :-) )
```suggestion
  return null;
```